internal/server/index.ts

```
@@ -105,56 +84,66 @@ function createRequestLogMessage(
-    onRequest,
-    preloadModulePath,
-    Shell,
-  }: {
-    onRequest?: OnRequestType;
-    preloadModulePath: string;
-    Shell: ReactShellComponent;
-  }) {
-    this.onRequest = onRequest;
-    this.preloadModulePath = preloadModulePath;
-    this.Shell = Shell;
+  }: BunextServerProps) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)